### DISCUSSION TITLE
I do  not have  Kibana  button  on my t-pot

### DISCUSSION THREAD

**Discussion by PLACIDEM** [telekom-security/tpotce#1431] [Q&A]
Hello Everyone,

I am new to t-pot and after t-pot installation succeeded  on my Azure VM, I realize that I do not  have  Kibana  access from my t-pot  portal. I can access Log  in /Data/ directory  but still  I need to visualize using Kibana. any idea how I can  add  or  integrate Kibana on my honeypot ?  
below  is the  Screenshot of my t-pot menu:

![image](https://github.com/telekom-security/tpotce/assets/96414710/ccc10a95-c686-4a04-a6af-159f63977262)

**Comment by t3chn0m4g3** [comment]
#1379
